internal/web/middleware.go

```diff
@@ -114,11 +114,13 @@ func cleanPath(pth string) string {
 func AuthMW(log *logrus.Entry, tm token.Manager) Middleware {
 	return func(next http.Handler) http.Handler {
 		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-
-			// signature validation
+			log.Info("Validating token!")
 			authz := r.Header.Get("Authorization")
 			parts := strings.Split(authz, " ")
 			if len(parts) != 2 {
+				if err := JSONErrorResponse(w, http.StatusUnauthorized, fmt.Errorf("invalid authz header")); err != nil {
+					log.WithError(err).Println("error creating json response")
+				}
 				log.Println("invalid authz header")
```

Comment:
Can this be changed to something like `log.Errorf("invalid authz header: %v", parts)`? We will get the error tag in the logs and will see the header content.

Reply:
fixed